### ISSUE TITLE
Create a Notebook for geocoding

### ISSUE THREAD

**Issue by dt-woods** [cga-wm/advgis-beta#16]
The process in the Geocoding Tutorial can be replicated in Notebooks.
The difference is that we can't use `geopy`, but we can use `arcpy` directly (thanks, Esri!). 
First, we need to set up a notebook with a copy of the basic code.
I think there's a link to the original script somewhere in the video.
Any help is good help.

**Comment by eosinski** [comment]
So I created a notebook and was able to get much of the absentee locations into a txt file: 

<img width="1257" alt="Future spreadsheet file" src="https://user-images.githubusercontent.com/67912370/87607902-677b8e00-c6cc-11ea-9d8e-b1d9fa5ae9ac.png">

However, I'm missing the lat and lon coordinates because I haven't figured out how to define geocode. I'm guessing I'm right at the part where using arcpy directly comes into play. After going through the website in the script, I'm still confused as to what a token is. In an attempt to set up a geocode function, I've done these steps in my notebook: 
![Attempting to Geocode](https://user-images.githubusercontent.com/67912370/87608153-2637ae00-c6cd-11ea-8e1a-16d85c531b26.png)

My notebook should be shared with the group. It's titled: EO_backup_geocoding_SP2. Any help or suggestions appreciated.

**Comment by dt-woods** [comment]
Doing some browsing on the web, I found this tutorial on [Understanding geocoders](https://developers.arcgis.com/python/guide/understanding-geocoders/). Some quick pointers on your notebook:

1. Remember to use `arcgis_online = GIS("home")` to make sure you are logged in with your account.
1. Rather than use `arcgis_online.content.search`, it looks like you can import the default geocoder, which should be fine for this application:

```py
from arcgis.geocoding import geocode
```

So all you need to do is build the query, for example:

```py
my_query = ", ".join([my_street, my_city, my_state])
```

and pass it to the geocode function imported from arcgis.geocoding

```py
my_result = geocode(my_query)
```

**Comment by parthree** [comment]
I got hung up on the token part too.

**Comment by dt-woods** [comment]
Yeah. Sorry about that. The token comes from an ArcGIS web application that I created in AGOL that acts like a key to gain access to the ArcGIS geocoder outside of Esri products. If you are in ArcGIS Notebooks and use the 

```py
gis = GIS("home")
```

then you don't need a special access token because you are already signed in with your W&M credentials and have direct access to all of their geoprocesses.
